jdaviz/configs/default/plugins/plot_options/plot_options.vue

```diff
@@ -182,7 +186,7 @@
       <plugin-switch
         :value.sync="axes_visible_value"
         label="Show axes"
-        api_hint="plt.axes_visible = "
+        api_hint="plg.axes_visible = "

```

Comment:
Impressive, I never would have caught a matplotlibian-slip like this.

Reply:
Cami actually had during a demo, so I made sure to do a full search for `plt.` in the code 😂